parlai/agents/transformer/modules.py

```diff
@@ -42,13 +46,16 @@ def _build_encoder(opt, dictionary, embedding=None, padding_idx=None, reduction=
         padding_idx=padding_idx,
         learn_positional_embeddings=opt.get('learn_positional_embeddings', False),
```

Comment:
why is this an opt.get? can we change it to opt['learn_positional_embeddings'] for consistency with the other args?